js/ui/drop_down_button.js

```diff
@@ -432,7 +432,8 @@ const DropDownButton = Widget.inherit({
     _popupHidingHandler() {
         this.option('opened', false);
 
-        this._setAriaExpanded(false);
+        this._setPopupButtonsAria(false);
+        this._setElementAria(false);
```

Comment:
Let's create a single method to update aria attributes to reduce a copypaste

Reply:
Refctored here: [08b3ae3acf750d1245b3bd08f0c6e468113ec5c1].